internal/impl/postgresql/input_postgrecdc.go

```diff
@@ -0,0 +1,562 @@
+		Example("postgres"),
+	).
+	Field(service.NewStringField("password").
+		Description("Password for the specified PostgreSQL user.")).
+	Field(service.NewStringField("schema").
+		Description("The PostgreSQL schema from which to replicate data.")).
+	Field(service.NewStringField("database").
+		Description("The name of the PostgreSQL database to connect to.")).
+	Field(service.NewStringEnumField("tls", "require", "none").
+		Description("Specifies whether to use TLS for the database connection. Set to 'require' to enforce TLS, or 'none' to disable it.").
+		Example("none").
+		Default("none")).
+	Field(service.NewBoolField("stream_uncomited").
+		Description("If set to true, the plugin will stream uncommitted transactions before receiving a commit message from PostgreSQL. This may result in duplicate records if the connector is restarted.").
+		Default(false)).
+	Field(service.NewStringField("pg_conn_options").
+		Description("Additional PostgreSQL connection options as a string. Refer to PostgreSQL documentation for available options.").
+		Default(""),
+	).
+	Field(service.NewBoolField("stream_snapshot").
+		Description("When set to true, the plugin will first stream a snapshot of all existing data in the database before streaming changes.").
+		Example(true).
+		Default(false)).
+	Field(service.NewFloatField("snapshot_memory_safety_factor").
+		Description("Determines the fraction of available memory that can be used for streaming the snapshot. Values between 0 and 1 represent the percentage of memory to use. Lower values make initial streaming slower but help prevent out-of-memory errors.").
+		Example(0.2).
+		Default(1)).
+	Field(service.NewIntField("snapshot_batch_size").
+		Description("The number of rows to fetch in each batch when querying the snapshot. A value of 0 lets the plugin determine the batch size based on `snapshot_memory_safety_factor` property.").
+		Example(10000).
+		Default(0)).
+	Field(service.NewStringEnumField("decoding_plugin", "pgoutput", "wal2json").
+		Description("Specifies the logical decoding plugin to use for streaming changes from PostgreSQL. 'pgoutput' is the native logical replication protocol, while 'wal2json' provides change data as JSON.").
+		Example("pgoutput").
+		Default("pgoutput")).
+	Field(service.NewStringListField("tables").
+		Description("A list of table names to include in the logical replication. Each table should be specified as a separate item.").
+		Example(`
+			- my_table
+			- my_table_2
+		`)).
+	Field(service.NewIntField("checkpoint_limit").
+		Description("The maximum number of messages of the same topic and partition that can be processed at a given time. Increasing this limit enables parallel processing and batching at the output level to work on individual partitions. Any given offset will not be committed unless all messages under that offset are delivered in order to preserve at least once delivery guarantees.").
```

Comment:
This description mentions partitions which is I assume copied from the kafka input?